sr_port/cache_stats.c

```diff
@@ -21,13 +21,15 @@ GBLREF	hash_table_objcode	cache_table;
 
 void cache_stats(void)
 {
-	int		total_attempts, ace;
+	long long	total_attempts, ace, cache_hits_ll, cache_fails_ll;
 	ht_ent_objcode 	*tabent, *topent;
 	cache_entry	*csp;
 
-	total_attempts = cache_hits + cache_fails;
+	cache_hits_ll = cache_hits;
+	cache_fails_ll = cache_fails;
+	total_attempts = cache_hits_ll + cache_fails_ll;
 	FPRINTF(stderr,"\nIndirect code cache performance -- Hits: %d, Fails: %d, Hit Ratio: %d%%\n",
```

Comment:
Since Hits and Fails will be 8-byte quantities (after addressing the previous comment), the %d should be changed to %lld for Hits and Fails.

Reply:
Changing the hit ratio formatting only to %lld, as that is currently the only 8-byte calculation.